src/subapps/admin/components/Settings/ViewsSubView.tsx

```diff
@@ -227,7 +219,7 @@ const ViewsSubView = () => {
   const { mutateAsync: handleReindexingAllViews, isLoading } = useMutation(
     restartIndexingAllViews,
     {
-      onError: error => {
+      onError: () => {
```

Comment:
it might be a good idea to capture the error in `Sentry`

Reply:
👍